tests/Ticket/gh110TestCase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+class Doctrine_Ticket_gh110_TestCase extends Doctrine_UnitTestCase
+{
+    public function testAddActAsColumnsToDocBlock()
+    {
+        $builder = new Doctrine_Import_Builder();
+        $class = $builder->buildDefinition(
+            array(
+                'className' => 'Ticket_gh110_TestRecord',
+                'topLevelClassName' => 'Ticket_gh110_TestRecord',
+                'is_base_class' => true,
+                'columns' => array(
+                    'id' => array(
+                        'type' => 'integer',
+                        'length' => 4,
+                    ),
+                    'my_custom_created_at' => array(
+                        'name' => 'created_at',
+                        'type' => 'my_custom_type',
+                        'length' => '',
+                    )
+                ),
+                'actAs' => array(
+                    'SoftDelete' => array(),
+                    'Timestampable' => array(
+                        'updated' => array(
+                            'disabled' => true,
+                        ),
+                        'unknown_column' => array()
+                    ),
+                    'UnknownActAs' => array(),
+                    // This template brings an already defined column
+                    'gh110_Template' => array(),
+                )
+            )
+        );
+
+        // We must replace the version as there is a timestamp in it.
```

Comment:
Should we create a new PR to get rid of that timestamp/version on the generated classes?  WDYT?

Reply:
For me, they can be dropped, yes. I'll have a look into it in the next days, if I find the time.

Reply:
While playing around with removing the timestamp/version, I noticed, it's a constant value and there is no need to replace/remove it in the test. The pr is updated but anyway, I think we can remove it.